static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -199,9 +200,6 @@ define([
      * attributes on the element.
      */
     function defineAdverts() {
```

Comment:
some of this terminology, particularly the verb usage, seems conflated now. `defineAdverts` just runs `queueAdvert` for every advert. Ideally, it should be consistent, like `createAdverts` and `createAdvert`, and companion functions like this might be more readable if they are adjacent to each other?